forms/collections/elements.js

```diff
@@ -109,13 +116,14 @@ define(function (require) {
       };
 
       errors = this.reduce(function(memo, elementModel){
+        var subFormErrors;
         if ( elementModel.get('type') === 'subForm' ){
-          memo = _.chain(elementModel.get('forms').invoke('getInvalidElements'))
-                  .compact()
-                  .pluck('errors')
-                  .flatten()
-                  .reduce(reducer, memo)
-                  .value();
+          subFormErrors = _.chain(elementModel.get('forms').invoke('getInvalidElements'))
+                            .compact()
+                            .pluck('errors')
+                            .flatten()
+                            .value();
```

Comment:
its always worth benchmarking. I didnt want to prematurely optimise it... and im not even convinced a single, easy to understand for loop can be written for subforms.